releasenotes/notes/deprecate-patch-py-methods-redis-requests-webbrowser-1b23da70723b764a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    All methods in redis/patch.py, rediscluster/patch.py, yaaredis/patch.py, requests/patch.py, and webbrowser/patch.py except patch() and unpatch() are deprecated and will be removed in version 3.0.0.
```

Comment:
Can we make a new release note entry per integration even though you're batching the PRs?  I think you can add more entries with more `- |`.